.github/workflows/spellcheck.yml

```
@@ -0,0 +1,27 @@
+name: Spell checking
+on:
+  pull_request_target:
+  workflow_dispatch:
+
+jobs:
+  spelling:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - uses: check-spelling/check-spelling@main
```

Souldn't we settle the version?
```suggestion
      - uses: check-spelling/check-spelling@v0.0.20-alpha5
```

---

I will put the last one, i.e v0.0.20-alpha7.
Thanks !